src/config.json

```
@@ -740,6 +740,7 @@
     "nfinity.space"
   ],
   "blacklist": [
+    "myetlhervwallet.com",
```

@randymetamask the domain shown in the tweet is `myewetlhervwallet.com` Notice the extra "we" in the middle there, and the L rather than a 1. For the 1 vs. L I did a whois check to see which was registered because it was ambiguous.
```suggestion
    "myewetlhervwallet.com",
```